physics/module_bl_mynn.F90

```diff
@@ -6226,7 +6591,7 @@ SUBROUTINE DMP_mf(                       &
         edmf_ent(K)=edmf_ent(K)+UPA(K,i)*ENT(K,i)
         edmf_qc(K) =edmf_qc(K) +UPA(K,i)*UPQC(K,i)
 #if (WRF_CHEM == 1)
```

Comment:
Is this dead code? Do we ever define WRF_CHEM=1 in FV3?

Reply:
It isn't used by me but maybe Ravan and Jordan use it. All WRF_CHEM if-defs will be removed soon as part of our ongoing work to "universalize" the MYNN to all dycores. This will be completed along with the merge with Laura Fowler's version of MYNN-EDMF. At that point, this WRF-CHEM code will be officially absorbed into the MYNN and probably regulated/protected by a new namelist option.